js/blocks/components/test/fetch-input.js

```diff
@@ -42,39 +39,56 @@ const baseProps = {
 
 describe( 'FetchInput', () => {
 	it( 'displays the value if there is no displayValue', () => {
-		const { input } = setup( { value, field: { label }, onChange: mockOnChange } );
+		const { input } = setup( {
+			value,
+			field: { label },
+			onChange: mockOnChange,
+		} );
 		expect( input.value ).toBe( value );
 	} );
 
 	it( 'displays the displayValue instead of the value if both are present', () => {
-		const { input } = setup( { value, displayValue, field: { label }, onChange: mockOnChange } );
+		const { input } = setup( {
+			value,
+			displayValue,
+			field: { label },
+			onChange: mockOnChange,
+		} );
 		expect( input.value ).toBe( displayValue );
 	} );
 
-	it( 'displays the popover when there are search results to show', async () => {
+	it( 'displays the popover when there are search results to show', () => {
 		const exampleResult = 'this-is-a-result';
 		const results = [ exampleResult ];
-		apiFetch.mockImplementationOnce( () => new Promise( ( resolve ) => resolve( results ) ) );
+		apiFetch.mockImplementationOnce(
+			() => new Promise( ( resolve ) => resolve( results ) )
+		);
 		const { input } = setup( baseProps );
 		fireEvent.focus( input );
 
-		const suggestion = await screen.findByText( exampleResult );
-		expect( suggestion ).not.toBe( null );
+		waitFor( () =>
+			expect( screen.getByText( exampleResult ) ).not.toBe( null )
+		);
 	} );
 
 	it.each( [
 		[ [], true ],
 		[ [ 'a-result' ], false ],
 		[ [ 'first-result', 'another-result' ], false ],
-	] )( 'should only have the error class if there are no results after focusing',
-		async ( apiResults, expected ) => {
-			apiFetch.mockImplementationOnce( () => new Promise( ( resolve ) => resolve( apiResults ) ) );
+	] )(
+		'should only have the error class if there are no results after focusing',
+		( apiResults, expected ) => {
+			apiFetch.mockImplementationOnce(
+				() => new Promise( ( resolve ) => resolve( apiResults ) )
+			);
 			const { input } = setup( baseProps );
 			fireEvent.focus( input );
 
-			await waitForDomChange( { container: input } ).then( () => {
-				expect( input.classList.contains( 'text-control__error' ) ).toStrictEqual( expected );
-			} );
+			waitFor( () =>
```

Comment:
This isn't related to e2e testing, but it fixes a failed test when I updated the `package.json` script versions.